services/src/atdd-staging/src/main/java/org/openkilda/atdd/staging/steps/FlowCrudSteps.java

```diff
@@ -138,6 +142,24 @@ public void flowsWithAlternatePaths(int flowsAmount, int alternatePaths, int bw)
         flowIsls.putAll(flowManager.createFlowsWithASwitch(flowsAmount, alternatePaths, bw));
         //temporary resaving flows before refactoring all methods to work with topologyUnderTest
         flows = flowIsls.keySet();
+        flows.forEach(flow -> flowPaths.add(northboundService.getFlowPath(flow.getId())));
+    }
+
+    @And("^flow paths? (?:is|are) changed")
+    public void flowPathIsChanged() {
+        Set<FlowPathPayload> actualFlowPaths = new HashSet<>();
+        flows.forEach(flow -> actualFlowPaths.add(northboundService.getFlowPath(flow.getId())));
+        assertThat(actualFlowPaths, everyItem(not(isIn(flowPaths))));
```

Comment:
This is not actually true. This may work in some cases, but some parts of the path may still remain the same after reroute, it is not guaranteed that _everyItem_ will be different

Reply:
Ok. Then we need to check only paths, not all items.